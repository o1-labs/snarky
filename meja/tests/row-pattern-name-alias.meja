let f = fun (x : [`A], y : [`B], b) => {
  switch (x, y, b) {
    | (x, _, true) | (_, x, false) => x
  };
};
