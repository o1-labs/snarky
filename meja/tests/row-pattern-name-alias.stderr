File "tests/row-pattern-name-alias.meja", line 3, characters 6-34:
Error: Incompatible types [`A] and [`B]:
       Cannot unify [`A] and [`B]: the constructor `A is not present in both.
